go/runtime/host/sandbox/bwrap.go

```diff
@@ -0,0 +1,154 @@
+package sandbox
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	dynlib "gitlab.com/yawning/dynlib.git"
+)
+
+const (
+	sandboxHostname = "oasis-core"
+
+	sandboxMountBinary = "/entrypoint"
+	sandboxMountLibDir = "/usr/lib"
+
+	bwrapPath = "/usr/bin/bwrap"
+)
+
+type bwrap struct {
+	*naked
+}
+
+// NewBubbleWrap creates a Bubblewrap-based sandbox.
+func NewBubbleWrap(cfg Config) (Process, error) {
```

Comment:
Something I found myself wishing for was a way to easily mount byte buffers as files in the container (`--ro-bind-data FD DEST`).

Reply:
Added the `BindData` option.